tests/TestCase.php

```diff
@@ -26,45 +28,54 @@ protected function getEnvironmentSetUp($app)
     {
         $this->initializeDirectory($this->getTempDirectory());
 
-        $app['config']->set('database.default', 'sqlite');
-        $app['config']->set('database.connections.sqlite', [
+        Config::set('database.default', 'sqlite');
```

Comment:
Let's use the `config()` helper instead of the facade. That's more in line with our other packages.